creating_images/custom.adoc

```diff
@@ -38,3 +40,14 @@ the information needed to proceed with the build:
 |`*DOCKER_SOCKET*`
 |This variable specifies the path to docker socket, if exposing docker socket was enabled on BuildConfig.
 |===
+
+== Custom builder workflow
+Although custom builder image author has a great flexibility in defining the build
+process on its own, still he should follow a few required steps necessary to seamlessly
+run a build inside of OpenShift. The required steps for a custom builder image are following:
+
+. Read the link:../rest_api/openshift_v1.html#v1-build[Build] definition, which
+contains all the necessary information about input parameters for the build.
+. Run the build process.
+. Push the image to the link:../rest_api/openshift_v1.html#v1-build[Build]'s output location if the
```

Comment:
you're still assuming the custom builder builds an image.  

this should be "if the product of the build is a new image, push the image to...."